doc/api/process.md

```diff
@@ -97,6 +97,60 @@ the child process.
 The message goes through serialization and parsing. The resulting message might
 not be the same as what is originally sent.
 
+### Event: 'multipleResolves'
+<!-- YAML
+added: REPLACEME
+-->
+
+* `type` {string} The error type. One of `'resolveAfterResolved'` or
```

Comment:
could just call type `'resolve'` or `'reject'`

Reply:
I see that this could be sufficient but I personally like the very explicit nature here. If you feel strongly about it, I'll change it, otherwise I would rather keep it as is.

Reply:
current naming is correct, there was a *resolve* *after* the promise *resolved*.

Reply:
Tanks, you're right. I didn't know "resolve" was also a noun.